.github/workflows/insiders.yml

```
@@ -120,6 +121,19 @@ jobs:
     name: Tests
     # The value of runs-on is the OS of the current job (specified in the strategy matrix below) instead of being hardcoded.
     runs-on: ${{ matrix.os }}
+    env:
+      # Something in Node 12.16.0 breaks the TS debug adapter, and ubuntu-latest bundles Node 12.16.1.
+      # We can remove this when we switch over to the python-based DA in https://github.com/microsoft/vscode-python/issues/7136.
+      # See https://github.com/microsoft/ptvsd/issues/2068
+      # At this point pinning is only needed for consistency. We no longer have TS debug adapter.
+      NODE_VERSION: 12.15.0
+      # Force a path with spaces and to test extension works in these scenarios
+      # Unicode characters are causing 2.7 failures so skip that for now.
+      special-working-directory: './path with spaces'
+      special-working-directory-relative: 'path with spaces'
```

What is the benefit of having a separate `'./path with spaces'` env variable, instead of using `'path with spaces'` everywhere?

---

So I don't mess up with the name. Just a variable replacement.

---

```suggestion
      special-working-directory: './path with 🌌s'
      special-working-directory-relative: 'path with 🌌s'
```
If we want to do some Unicode path and spaces, this might help?

---

Why not use `env.special-working-directory-relative` everywhere, instead of having `env.special-working-directory` and `env.special-working-directory-relative`?

---

Cause they're different? I wasn't sure how to concat a string in the yaml file so I just made to variables.

---

Adding something with Unicode like Karthik suggested is probably a good idea.